README.md

```diff
@@ -43,13 +45,15 @@ The stryker-jest-runner also provides a couple of configurable options using the
 }
 ```
 
-| option | description | default value |
-|----|----|----|
-| project (optional) | The type of project you are working on. Currently "react" and "default" are supported. When "react" is configured, "react-scripts" is used (for create-react-app projects). When "default" is configured, your "config" option is used. | default |
-| config (optional) | A custom jest configuration (you can also use `require` to load your config here) | undefined |
+| option | description | default value | alternative values |
+|----|----|----|---|
+| project (optional) | The type of project you are working on. | `default` | `default` uses the `config` option (see below)|
+| | | | `react` when you are using [create-react-app](https://github.com/facebook/create-react-app) |
+| | | | `react-ts` when you are using [create-react-app-typescript](https://github.com/wmonk/create-react-app-typescript) |
+| config (optional) | A custom Jest configuration object. You could also use `require` to load it here) | undefined | |
 
-**Note:** When neither of the options are specified it will use the jest configuration in your "package.json". \
-**Note:** the `project` option is ignored when the `config` option is specified. \
+**Note:** When neither of the options are specified it will use the Jest configuration in your "package.json". \
+**Note:** the `project` option is ignored when the `config` option is specified.
 **Note:** Stryker currently only works for CRA-projects that have not been _ejected_.
```

Comment:
What does _ejected_ mean? Is it a React thing? Maybe we can add a link to where the concept is explained.

Reply:
See the [CRA documentation](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-eject). This particular remark was already there since #46.

Basically, CRA allows you to "eject", which
> will copy all the configuration files and the transitive dependencies (Webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except eject will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.